net/common/src/main/java/de/danoeh/antennapod/net/common/RequestHeaderIntercepter.java

```diff
@@ -0,0 +1,37 @@
+package de.danoeh.antennapod.net.common;
+
+import androidx.annotation.NonNull;
+
+import okhttp3.Interceptor;
+import okhttp3.Response;
+
+import java.io.IOException;
+import java.util.Locale;
+
+public class RequestHeaderIntercepter implements Interceptor {
+    public static final String ACCEPT_ENCODING = "gzip, identity";
+
```

Comment:
This should actually be `gzip;1.0, identity:0.1` according the HTTP spec if we wanted to prioritize `gzip`